### ISSUE TITLE
How to specify multiple authors?

### ISSUE THREAD

**Issue by timotk** [binxio/wordpress-markdown-blog-loader#6]
- Multiple authors is possible in Wordpress
- When downloading an existing post with multiple authors to markdown, there is only one author in the markdown frontmatter.
- How can we specify multiple authors in markdown?

**Comment by mvanholsteijn** [comment]
Multiple authors is not yet supported. I will fix that when I get back for holiday

**Comment by Roymprog** [comment]
I was looking into adding this as I wanted to add a multi-author blog to the xebia-blog repo. 
For multiple authors you need to get the id for multiple users, see [this code](https://github.com/binxio/wordpress-markdown-blog-loader/blob/1.3.3/src/wordpress_markdown_blog_loader/api.py#L324). I'm not allowed to because listing users is only allowed as [Administrator](https://wordpress.org/documentation/article/roles-and-capabilities/#administrator).
I tried to work around it by manually providing the user id. However, wordpress does not support multiple authors by default. We are using the [Co-Authors-Plus](https://github.com/Automattic/Co-Authors-Plus/wiki) plugin to enable multi-author. This means we cannot create/update a blog post with multiple authors via the Wordpress REST API. 
AFAIK, the plugin also doesn't support creating/updating a post with multiple authors either.

**Comment by mvanholsteijn** [comment]
@Roymprog yes I reached that conclusion as well. Wordpress is an insanely popular and inept system..

You would need to make PR on the Co-Authors-Plus plugin to support multiple authors.

To complicate matters, the coauthor user id is not the user id of the user in WP.

**Comment by ramonvermeulen** [comment]
Without multiple author support I assume the way to go is:

1. Publish the blog on one author
2. Add the second author via the WP UI?

Only downside I can imagine is that the og-banner for socials is only generated with one of the two photos/names.
